inventory/host_vars/openfoodnetwork.com.br/config.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+domain: openfoodnetwork.com.br
+rails_env: production
+
+admin_email: frattezi@tremtec.com
+
+mail_domain: openfoodnetwork.com.br
+
+swapfile_size: 2G
+unicorn_timeout: 120
```

Comment:
I'd say this should be lower to start with, we have large timeouts because of some reports in some servers with a lot of data.

Reply:
Understood thank you!